tests/conftest.py

```diff
@@ -96,9 +96,9 @@ def dockerfile(request):
     return request.config.getoption('--dockerfile')
 
 
-@pytest.fixture(scope='session')
```

Comment:
why it should be on the class level? I think it is unique for the session.

Reply:
'image' and 'tester' are used inside test functions and must be assigned before use. I put them in a class to call with self without declaring in function parameters.